catalog.html

```diff
@@ -56,23 +56,201 @@
+                            <input class="input-hidden form-radio__input" type="radio" name="mix" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Mix</span>
+                            </span>
+                        </label>
+                        <label class="form-radio form-radio--col">
+                            <input class="input-hidden form-radio__input" type="radio" name="orange" required>
+                            <span class="form-radio__inner">
+                                <span class="form-radio__box"></span>
+                                <span class="form-radio__text">Orange</span>
+                            </span>
+                        </label>
+                    </div>
+                </details>
+                <button type="submit" class="btn btn--color btn--full">Apply</button>
+            </form>
         </aside>
 
         <article class="blog-content">
 
             <section class="catalog catalog--big">
+                <div class="catalog-filters">
+                    <h2 class="subtitle subtitle--left">All item</h2>
+                    <div class="catalog-controls">
+
+                        <form class="form-select" action="#" method="get">
+                            <select class="select" name="sort" id="sort">
+                                <option class="select__option" value="">Sort by</option>
+                                <option class="select__option" value="newest">Newest</option>
+                                <option class="select__option" value="popular">Most popular</option>
+                                <option class="select__option" value="increasing">Increasing in price</option>
+                                <option class="select__option" value="lowering">Lowering the price</option>
+                            </select>
+                        </form>
+
+
+                        <form class="search-form search-form--catalog" action="/search" method="get">
+                            <button type="submit" class="search-form__btn">
+                                <svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+                                    <path d="M21 21L16.65 16.65M19 11C19 15.4183 15.4183 19 11 19C6.58172 19 3 15.4183 3 11C3 6.58172 6.58172 3 11 3C15.4183 3 19 6.58172 19 11Z" stroke="#F5F8FB" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/>
+                                </svg>
+                            </button>
+                            <input class="search-form__input" type="search" placeholder="Search NFT" aria-label="Search"> 
+                        </form >
+                    </div>
+                </div>
+                <div class="catalog-tags">
```

Comment:
It's a filter buttons.
Structure:
```
<div class="filter-buttons">
  <button type="button">NFT<svg>'x'</svg></button>
  ...
  <button  type="reset">Clear all</button>
</div>
```

Use it instead div.catalog-tags + ul.tags__wrapper

Reply:
fixed. done